mobilitypath_publisher/src/mobilitypath_publisher.cpp

```diff
@@ -0,0 +1,138 @@
+/*
+ * Copyright (C) 2019-2020 LEIDOS.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+
+#include "mobilitypath_publisher.h"
+
+namespace mobilitypath_publisher
+{
+    
+    MobilityPathPublication::MobilityPathPublication():
+                            x_threshold(20.0),
+                            y_threshold(3.0) {}
+
+    void MobilityPathPublication::initialize()
+    {
+        nh_.reset(new ros::CARMANodeHandle());
+        pnh_.reset(new ros::CARMANodeHandle("~"));
+        pnh_->param<double>("trajectory_threshold_x", x_threshold, 20.0);
+        pnh_->param<double>("trajectory_threshold_y", y_threshold, 3.0);
```

Comment:
These parameter names do not match the names in your config file.

Reply:
fixed